### ISSUE TITLE
Some strings return invalid hex values

### ISSUE THREAD

**Issue by jakeleboeuf** [stuneak/create-color#2]
### tl;dr
Some strings return invalid hex values  🤷‍♂ 

### More Details

I'm using `user.id`'s to generate color values. These are `uuid`'s and generally look like `9fc91124-0c99-48f8-863e-49f173ead3f7` as you'd expect. This is typically fine- and causes no issues. However, some `uuid`'s return invalid hex values (like the one I'm using here).

```js
import createColor from "create-color";

const background = createColor("9fc91124-0c99-48f8-863e-49f173ead3f7");
```

### Demo
I've done a _tiny_ amount of work to reproduce the issue, but not enough to understand the problem. [Here's a demo](https://09tk1.csb.app/) with some of the evil strings.

**Comment by stuneak** [comment]
@jakeleboeuf Thank you! I will try to find a mistake within 3 days and fix it!

**Comment by stuneak** [comment]
@jakeleboeuf  Thank you!

I fixed it in version 2.0.1.
